### DISCUSSION TITLE
Trained model generates bad pictures

### DISCUSSION THREAD

**Discussion by SirSatorik** [TheLastBen/fast-stable-diffusion#2859] [Q&A]
Hi all! Now I trained a model to generate Minecraft-style pictures and I have the following problem:
I write a prompt for a finished model, but it generates nonsense

Data:
dataset of 50 photos
description of 10 words
text encoder 450 steps and unet 1500
trained on 512x512 pictures and also generated
taught with pictures like this:
![image-003](https://github.com/TheLastBen/fast-stable-diffusion/assets/108075233/2aa52636-4776-41fb-b255-7a000ec9a3ce)

In the end it produced this:
![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/108075233/a18edf80-27e5-4252-b791-42a6266c5c93)

Tested using SDGUI 1.12.02b
![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/108075233/7da54bfa-34b8-4f4c-9e62-89e609728883)

**Comment by Ibxx0403** [comment]
I am not an expert, but do you use offset noise? it is apparently better at training for stylised content like this. Also it could be the prompt you are using.  Another thing is that SD is pretty bad at text, so its probably not a good idea to try getting it to write text. But ultimately its very hard to decipher what the issue is when i dont know the prompts, images and captions used.
